ledger/catchpointtracker_test.go

```diff
@@ -1554,8 +1679,8 @@ func TestCatchpointFastUpdates(t *testing.T) {
 			ml.trackers.committedUpTo(round)
 		}(i)
 	}
-	ml.trackers.waitAccountsWriting()
 	wg.Wait()
+	ml.trackers.waitAccountsWriting()
```

Comment:
this prevents calling `accountsWriting.Wait` before calling any `accountsWriting.Add`. Note these are called from different goroutines and could lead to a data race. Golang pattern is to have Add/Wait called from the same goroutine.